Jenkinsfile

```diff
@@ -268,7 +268,7 @@ pipeline {
                     }
                 }
                 stage('OpenCL GPU tests') {
-                    agent { label "gelman-group-win2 || linux-gpu" }
+                    agent { label "inux-gpu" }
```

Comment:
This might be an unintended typo.

Reply:
Yeah, silly mistake :) Thanks. Should be good to approve now.